src/add-score.js

```diff
@@ -0,0 +1,19 @@
+import { addInfo } from './display-table';
+
+function sendData(name, score) {
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/EO65suUhDU47wLiHboGe/scores/', {
+    method: 'POST',
+    body: JSON.stringify({ user: name, score }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.json());
+}
+
+export default function addScore() {
+  const name = document.querySelector('#name').value;
+  const score = document.querySelector('#score').value;
+  sendData(name, score);
+  addInfo(name, score);
+}
```

Comment:
- [ ] Kindly ensure `arrow functions` are used instead of the `function` keyword. Javascript's ES6 brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more. ES6 introduces us to many great features like `arrow functions`. The arrow function is really awesome and makes your code more readable, more structured, and look modern. Take a minute and also go through this [article](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html) on why ES6 arrow functions are so awesome